nimsuggest/tester.nim

```diff
@@ -279,7 +279,13 @@ proc runEpcTest(filename: string): int =
         os.sleep(50)
         inc i
       let a = outp.readAll().strip()
-    let port = parseInt(a)
+    var port: int
+    try:
+      port = parseInt(a)
+    except ValueError:
+      echo "Error parsing int: " & a
+      echo outp.readAll()
+      raise
```

Comment:
What does that do? Improve the error message but does not handle the error (it uses re-raising)? But who had problems with the previous error message?

Reply:
During the development of this feature, I had a nimsuggest crash that only happened on the GitHub CI machines, which I could not reproduce on my computer. Therefore, I added this, so that the stack trace, produced by nimsuggest is printed on the screen, and becomes visible in the GitHub CI log. The bug, that caused the crash has been fixed, but I left this here, because it's useful for diagnostic purposes, even though it's unrelated to the inlay hints for exceptions. I can extract it to a separate PR if you want?

Reply:
Ideally you simply remove it as I don't like it. If you want a better/different error mesage handle the error completely aka replace the `raise` by `quit`.

Reply:
Done: https://github.com/nim-lang/Nim/pull/23202/commits/a7bf527cde473f1f7be17ebe3ac17a38c948e199